### ISSUE TITLE
Template for future Readmes

### ISSUE THREAD

**Issue by jarumihooi** [clamsproject/aapb-annotations#46]
### Because

we want the project readmes to all have the same structure (and they seem to be adaptable that), we will create a template for the readmes that each readme will follow. 

Currently, the main issue is the discussion of the category names with @keighrim 

We will use this branch's [slate readme](https://github.com/clamsproject/aapb-annotations/blob/23-slates-readme/january-slates/readme.md) as the guide for the template. 

====
Question: 
Can we have optional or differences in formatting? 
```
# Official Project Name (Caps no dashes)

## Project Overview

### (Domain Knowledge Term Explanation optional?) <- right here, is this ok as optional?
About the project goals.

### Specs
* Annotation project name
```

### Done when

- [x] create and discuss the template (incl. category names) 
- [x] Modify existing and inprogress readmes to conform to template (softly, without coded validation).

**Comment by keighrim** [comment]
This might be a good starting point; 
``` bash 
$ grep -P "^#+ " january-slates/readme.md 
# Slates
## Project Overview
### What is a film slate? 
### Specs
## Tool Installation: None
## Annotation Guidelines
### Preparation
### What to Annotate
### How to Annotate It 
### Decisions, Differentiations, and Precision during Annotation
## Data format and `process.py`
### `raw` data
### [`process.py`](process.py)
### `golds` data
## See also 
### 2020 Evaluation Dataset   

```

**Comment by keighrim** [comment]
and here's the template file from the previous PR. https://github.com/clamsproject/aapb-annotations/pull/47/commits/9904589039b142cfb0028ee32c03601faf1bc495 .

**Comment by jarumihooi** [comment]
Thanks I have a copy as well, is there something you want done with the grep'd version? 

Still have this open question also: 
Can we have optional topic-lines? 
eg. `what is a slate/phenomenon?` -> optional?

**Comment by keighrim** [comment]
Unless we automatically "validate" the readme files based on some formal language (XML, jsonscheme, etc), I don't think we can actually _force_ writers to use very specific sections or their title. For now I'm considering this template as more like a general guideline, rather than a hard restrictions.

**Comment by jarumihooi** [comment]
Ive updated the wording on the issue to specify that the current and inprogress readmes should be changed to conform to this (as a guideline). 

I believe the recently merged issue #53 fixes the first bullet. Manually updating.

**Comment by jarumihooi** [comment]
The second bullet was handled piecemeal by commits and revisions to the readmes over a few PRs: 
https://github.com/clamsproject/aapb-annotations/pull/69/commits/03672c8b36b13a62b8f2f4ad7a9bd9f8f515b4fb from #69 , #60 , #64 , #59 , #55
